docs/gulp/tasks/jshint.js

```diff
@@ -0,0 +1,7 @@
+var gulp = require('gulp');
+var shell = require('gulp-shell');
+var handleErrors = require('../util/handleErrors');
+
+gulp.task('jshint', shell.task([
+  '../node_modules/.bin/jsxhint --harmony ../src/* ../src/**/* ../src/**/**/* ./src/app/* ./src/app/**/* ./src/app/**/**/* ./src/app/**/**/**/* --exclude ../src/utils/modernizr.custom.js'
```

Comment:
I know these successions of similar globbing patterns look ugly, but I haven't found a way to make them more general. I'll be glad if anyone lets me know of a solution.